_data/cezarmathe/cezarmathe.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Armand Cezar Mathe
+institution: "Andrei Muresanu" National College, Dej, Romania
+profile_pic: cezarmathe.png
+quote: "Underpromise and overdeliver."
```

Comment:
Btw this will appear as just `Underpromise and overdeliver.`, and not `"Underpromise and overdeliver."`. 

You may want to surround this line with additional (single) quotes:

```suggestion
quote: '"Underpromise and overdeliver."'
```

Reply:
Hi. You're not looking at the latest version. The most recent changes remove all double quotes from the markdown file. Also, as I've said before, my picture is actually a jpeg, not a png. I do not know why @juanpflores changed the file name.

Reply:
oops

Reply:
btw i don't work for github so i can't help you with the jpeg thing

Reply:
oh, got it.